src/encoded/static/vis_defs/STARR_vis_def.json

```diff
@@ -0,0 +1,78 @@
+{
+    "STARR": {
+        "rule": "'{assay_term_name}' == 'STARR-seq'",
+        "assay_composite": {
+            "longLabel":  "Collection of ENCODE STARR-seq experiments",
+            "shortLabel": "ENCODE STARR-seq"
+        },
+        "longLabel":  "{assay_title} of {replicates.library.biosample.summary} - {accession}",
+        "shortLabel": "{assay_title} of {biosample_term_name} {accession}",
+        "sortOrder": [ "Biosample", "Replicates", "Views" ],
+        "Views":  {
+            "tag": "view",
+            "group_order": [ "Peaks", "Signal of unique reads" , "Signal p-value", "Control normalized signal" ],
+            "groups": {
+                "Peaks": {
+                    "tag": "aPKS",
+                    "visibility": "dense",
+                    "type": "bigBed",
+                    "output_type": [ "peaks" ]
+                },
+                "Signal of unique reads": {
+                    "tag": "bSIG",
+                    "visibility": "hide",
+                    "type": "bigWig",
+                    "viewLimits": "0:10",
+                    "autoScale": "off",
+                    "maxHeightPixels": "64:18:8",
+                    "windowingFunction": "mean+whiskers",
+                    "output_type": [ "signal of unique reads" ]
+                },
+                "Signal p-value": {
+                    "tag": "cSPV",
+                    "visibility": "hide",
+                    "type": "bigWig",
+                    "viewLimits": "0:10",
+                    "autoScale": "off",
+                    "maxHeightPixels": "64:18:8",
+                    "windowingFunction": "mean+whiskers",
+                    "output_type": [ "signal p-value" ]
+                },
+                "Control normalized signal": {
+                    "tag": "dCNS",
+                    "visibility": "full",
+                    "type": "bigWig",
+                    "viewLimits": "0:10",
+                    "autoScale": "off",
+                    "maxHeightPixels": "64:18:8",
+                    "windowingFunction": "mean+whiskers",
+                    "output_type": [ "control normalized signal" ]
+                }
+            }
+        },
+        "other_groups":  {
+            "dimensions": { "Biosample": "dimY", "Replicates": "dimA" },
+            "dimensionAchecked": "first",
```

Comment:
Just to confirm, we decided to show only the first rep by default? Also which output types are `full`, `dense` and `hide` is decided?

Reply:
I think most experiments showed only the first rep by default, so I went with that convention. We can show all if that's better? I don't really have a strong preference either way. 

For the bigBed (peaks) file I figured that it would be good to have it viewed as dense - track as a single line. The other bigWig (signal) files - the decision to use full or hide was just determined in that I thought probably most users would be most interested in the control normalized signal which would have full visibility vs. the other signal files (hide).